### ISSUE TITLE
Empty table layout gives exception at start

### ISSUE THREAD

**Issue by mennodeij** [picoe/Eto#259]
Adding an empty table layout gives an exception when running the program due to usage of LINQ Max() extension, which cannot run on empty collection.

```
        public class SimpleForm : Form
        {
            public SimpleForm()
            {
                Content = new TableLayout();
                ClientSize = new Size(400,400);
            }
        }


        [STAThread]
        static void Main()
        {
            
            Application app = new Application(Eto.Platforms.Wpf);
            app.Run(new SimpleForm());
        }
```

Stacktrace
```
   at System.Linq.Enumerable.Max(IEnumerable`1 source)
   at System.Linq.Enumerable.Max[TSource](IEnumerable`1 source, Func`2 selector)
   at Eto.Forms.TableLayout.Create()
   at Eto.Forms.TableLayout.OnDeserialized(Boolean direct)
   at Eto.Forms.TableLayout.OnLoad(EventArgs e)
   at Eto.Forms.Control.TriggerLoad(EventArgs e)
   at Eto.Forms.Container.OnLoad(EventArgs e)
   at Eto.Forms.Form.Show()
   at Eto.Forms.Application.<>c__DisplayClass3.<Run>b__2(Object sender, EventArgs e)
   at Eto.PropertyStore.TriggerEvent[T](Object key, Object sender, T args)
   at Eto.Forms.Application.OnInitialized(EventArgs e)
   at Eto.Forms.Application.Callback.<>c__DisplayClass9.<OnInitialized>b__8()
   at Eto.Platform.Invoke(Action action)
   at Eto.Forms.Application.Callback.OnInitialized(Application widget, EventArgs e)
   at Eto.Wpf.Forms.ApplicationHandler.Run()
   at Eto.Forms.Application.Run(Form mainForm)
```

**Comment by cwensley** [comment]
This should already be fixed in the develop branch (not yet pushed to nuget).  You can add the development feed (https://www.myget.org/F/eto/) to your nuget sources list to give it a try!

**Comment by cwensley** [comment]
Double checked, and yes indeed this is fixed in the 2.1 develop branch.  Thanks for reporting the issue!
